src/main.coffee

```diff
@@ -173,6 +182,8 @@ getCss = (variables, styles, verbose, done) ->
 
       try
         css = result.css
+        folderPath = path.dirname compiledPath
+        fs.mkdirSync folderPath, {recursive:true}
```

Comment:
Ensure the folder exists in the memory filesystem before attempting to write files